css/amp.css

```diff
@@ -219,10 +219,10 @@ i-amphtml-sizer {
  * does not allow activation of buttons.
  */
 .i-amphtml-screen-reader {
-  position: fixed !important;
-  /* keep it on viewport */
-  top: 0px !important;
-  left: 0px !important;
+  position: absolute !important;
+  /* keep it on viewport. TalkBack likes this >=12px */
+  top: 12px !important;
```

Comment:
Strangely yes. After your comment I noticed it was partially hidden because of the `transform: translateY(-50%)` property on `.amp-carousel-button`, but even with that overridden to `none`, TalkBack will not click on it with `top` and `left` values below `12px`.
I'll add the override for transform, but unless we know of a downside to having a value above `0px` it seems like a safe choice. It does suck having it be a magic number though.